File: core/src/test/resources/fixtures/manifest-groovy-java/app/src/main/java/com/example/oldname/MainActivity.java
package com.example.oldname;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import com.example.oldname.util.Helper;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Helper helper = new Helper();
        helper.tag();
    }
}

File: core/src/test/resources/fixtures/kts-kotlin-no-namespace/app/src/main/kotlin/com/example/oldname/MainActivity.kt
package com.example.oldname

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.oldname.util.Helper

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Helper.show()
    }
}
